.github/workflows/debian.yml

```diff
@@ -0,0 +1,54 @@
+name: Build debian release
```

Comment:
Any particular reason why this is a separate file? IMO this should reside inside the release.yml as the release action will create and publish a new tag + assets. Having two entry points would mean two different releases(which is not the case). We need the deb file to be a part of our release itself.

Reply:
Should I add it in the release.yml file then? with a separate job (because we need the binary name as "deb" for dynamic 
 version name)

Reply:
@AviralVerma13 checkout the video we recorded https://youtu.be/ukiLoZIygPM.